base/src/main/kotlin/net/corda/v5/base/annotations/VisibleForTesting.kt

```diff
@@ -1,6 +1,10 @@
 package net.corda.v5.base.annotations
 
-/** The annotated object would have a more restricted visibility were it not needed in tests. */
+/** The annotated object would have a more restricted visibility were it not needed in tests.
```

Comment:
Not sure what we're saying here..

"The annotated object has a more restricted visibility if not needed in tests." ?
"If not needed in tests, annotate an object to apply more restricted visibility ."

Reply:
We are saying that we would like the object annotated with this annotation to have private or package visibility, but this can't be done because it would break some tests that need access to test this object.